### PR TITLE
Portfolio: mobile version

### PR DESCRIPTION
**Pull request by Loltolo-Lesapiti**
This project builds the first two sections of the portfolio mobile version website.

- The toolbar or the header and 
- The headline section, right after the header.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--Navigation menu-->
+    <nav>
+        <div class="logo">
+            <a href="#">My logo</a>
+        </div>
+        <i class="fa fa-bars"></i>
+    </nav>
```
Comment (tufoinnkuo10) [inline index.html:19]: - [ ] kindly put the navigation section in a `header tag`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--Navigation menu-->
+    <nav>
+        <div class="logo">
```
Comment (tufoinnkuo10) [inline index.html:15]: - [ optional] you can use your name here in the absence of a logo

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="#">My logo</a>
+        </div>
+        <i class="fa fa-bars"></i>
```
Comment (tufoinnkuo10) [inline index.html:18]: - [ ] please only the icon should be inside the nav bar and not the whole header section

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    </nav>
+    <!--Home section-->
+    <section class="home">
+        <div class="home-content">
+            <h1>I'm Petro <br> Glad to see <br> you!</h1>
```
Comment (tufoinnkuo10) [inline index.html:23]: - [ ] kindly use the same font family here('Poppins') and font weight (700)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Loltolo-Lesapiti 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by muneebulrehman — APPROVED** [review]
Hi @Loltolo-Lesapiti , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good followup with previous reviewer
- Good PR description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**